### DISCUSSION TITLE
All GitHub Actions pipelines failing with "this job failed" message

### DISCUSSION THREAD

**Discussion by soungsid** [community/community#53164] [Actions]
### Select Topic Area

Bug

### Body

Hello,

I hope this message finds you well. I am writing this support ticket to report that I have been experiencing issues with all of my GitHub Actions pipelines. Whenever I try to run any pipeline, I receive the error message "this job failed", without any further explanation.

I would greatly appreciate it if you could help me investigate this issue further and provide guidance on how to resolve it. I rely heavily on GitHub Actions for my development workflow, and this issue is causing significant disruption to my work.

Please let me know if you need any additional information from me to assist with troubleshooting. I look forward to hearing from you soon and resolving this issue promptly.

Thank you for your attention to this matter.

**Comment by ganzenman** [comment]
hi, we see the same error today with no obvious reasons. i'll keep an eye on this thread to see if others have the same issue

**Reply by ganzenman** [reply]
issue found, protection rules were in place that prevent an "apply"
hidden deep in the gui we found the error, but not in the obvious main screen

**Reply by springbootml** [reply]
Hi, 
I don't really understood. Is the issue caused by a setting ? 
If you found the solution can you share it with us? 
thank you

**Reply by ganzenman** [reply]
in our case the issue was a bit of both worlds:
we have environment settings that specify the deployment branches. if you don't follow the correct branch order it fails
secondly, there was an underlying github runner issue on the kubernetes cluster which also caused failures. these runners are fixed and all the flows are healthy again

**Reply by lebenitza** [reply]
Having protection rules on envs that also specify certain branches was my problem as well. GitHub actions should make it more obvious in this case that the protection rule does not allow it to run.

**Comment by jtraub91** [comment]
Seeing this issue as well. Previously successful workflows are now showing "this job failed" with no further info, when re-run.

**Reply by jtraub91** [reply]
Thought it might be a usage limit issue so I reduced the parallel jobs to 5, but this did not fix the issue.

**Reply by jtraub91** [reply]
Welp, must've been a usage limit since switching to a public repo resolved my issue. Open-source FTW 🙌

**Reply by maziyarpanahi** [reply]
> Thought it might be a usage limit issue so I reduced the parallel jobs to 5, but this did not fix the issue.

Where do you set this parallel number, I don't have any such thing in my settings. (I thought the jobs should be queued instead of failing, but they just say `This job failed`)

**Reply by jtraub91** [reply]
@maziyarpanahi it wasn't really a setting per se, I just reduced the amount of jobs by reducing the matrix of OS versions to run on in the workflow. The symptoms sound the same as what I was experiencing, but I still don't know 100% the root cause. It seems like you might be running into a [usage limit](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits).

**Reply by maziyarpanahi** [reply]
Thanks a lot @jtraub91 
It seems so, I really thought it would queue anything above that concurrent limit, but it seems some of them just fail without being queued. 
Thanks again, this was really helpful

**Comment by frindly** [comment]
This started happening to my jobs on several repositories starting last night, we're on the enterprise plan and way, way below the usage limit.  Is there something going on w/ runners?  I'm also seeing jobs just stuck in the "Queued" status.

**Reply by riteshbxr** [reply]
I am facing the same right now... started 3-4 hours ago. Seeing this randomly in few actions

**Reply by nilscam** [reply]
Same here

**Comment by devanup** [comment]
https://www.githubstatus.com/

If you go to this website it shows a list of issues they're facing. I think it's safe to say that Github isn't functioning properly right now.

**Comment by bhaller** [comment]
I'm getting this sometimes too, for no apparent reason, on things that previously built fine.

Partly this is a matter of transient outages, etc., which is of course understandable.  But there is a larger issue here, that displaying "This job failed" with no further explanation is singularly unhelpful.  It would be great if it could provide a more meaningful error message that makes it more clear whether the error is in something I did, or is an internal problem that I should just ignore.  So the poor error message is a bug that I'd like to report.  Anybody know where I might file an issue on this?  I did a bit of googling and ended up here.  :->

**Comment by danilodesolesonar** [comment]
Is anyone with "This job failed" again? experiencing it now just from a single repository... Additionally, the GH status page doesn't show any major outages.

**Reply by athira-1142** [reply]
This is happening from one repository. Is there any solution for the same.

**Comment by sergejs-katusenoks** [comment]
We are experiencing this since the morning (BST time). Random actions are failing (even within the same repo) and status is "This Job Failed". Action runs for 0 seconds, so it probably does not run at all.

**Reply by sergejs-katusenoks** [reply]
We have found out where the issue was. We have updated an OIDC claim and included "Environment". Since it does not exist in all jobs, GitHub runner fails and blows up behind the scenes without showing any logs. All our jobs that have had set a permission "id-token: write" (required for OIDC connection) were failing with no logs. 

If one encounters such error again I would recommend removing all CI permissions for the sake of test. Hopefully our experience helps someone.

Would be great if GitHub could provide more details on such errors.

**Reply by achernev** [reply]
Thank you @sergejs-katusenoks, my called workflow requires `id-token: write`, adding the permission to do so in the calling workflow fixed it for me.
```yaml
permissions:
  id-token: write
```

**Comment by emmyoop** [comment]
We have also started to get this screen on existing workflows.  I've noticed a pattern that we get the empty "This Job Failed" message when the job fails due to exceeding the maximum execution time.  In the past the web showed those details of why it failed.  Using the gh cli you can still get to those details but it would be great if it showed up in the web again.

**Reply by lakim** [reply]
We had `This Job Failed` a lot and absolutely no information on the Web UI.
Thanks for pointing out to the gh cli @emmyoop!
Running `gh run view --job JOB_ID` gave us a proper error message. In our case we were over our spending limits.

**Reply by sebabelmar** [reply]
Thanks!!! I had to pay for more Github Accion minutes.
`gh run view JOB_ID` saved me. And the email I did not see.

**Reply by ceciliarivero** [reply]
Thank you! That worked! 🎉 

In case this is helpful to others, these are the steps I followed:

```
brew install gh

(authenticate with GitHub)

// See the list of workflows to get the workflow_id of the one that's failing:
gh run list

// Get the job_id
gh api repos/:owner/:repo/actions/runs/<workflow_id>/jobs --jq '.jobs[] | {id, name, status, conclusion}'

// See the job's log
gh run view --job <job_id>
```

Update: You can actually see more information by going to the Summary section, these steps ☝️ might not be needed.
